examples/eris/src/commands/help.js

```diff
@@ -0,0 +1,23 @@
+const fs = require("fs");
```

Comment:
```suggestion
const fs = require("node:fs");
```

https://nodejs.org/api/modules.html#core-modules